github/ci/prow-deploy/files/jobs/kubevirt/macvtap-cni/macvtap-cni-presubmits.yaml

```diff
@@ -13,13 +13,13 @@ presubmits:
         grace_period: 5m
       max_concurrency: 6
       labels:
-        preset-dind-enabled: "true"
+        preset-podman-in-container-enabled: "true"
```

Comment:
@oshoval for the sake of my sanity, would you explain (or point me to the docs) what these two knobs do ? `preset-dind-enabled` / `preset-podman-in-container-enabled`

Reply:
Those are preset that config the bootstrap image with the env vars for example that are needed
for example the new one configures an env var that tells the bootstrap image to create podman socket, volume and so on

https://github.com/kubevirt/project-infra/blob/042be96d3927f12b02d383e95561cf03b6c7dd9d/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L434

Reply:
Correct if I'm wrong; this is needed because the "lanes" themselves run in containers ?... And this setting would make them run in podman containers, not docker containers.

Reply:
The lanes runs in a container, that use bootstrap image
we are using here a different image, this image need a specific volume and env var in order to run podman (in docker*) correctly
before that we had other image that needed other env var and other mount in order to use docker (in docker*)
So both the image and the label is needed, the images do not have both docker and podman
(note that them have docker / docker socket which is an alias to podman and its socket, but only an alias)

(The CI external CRI that runs on CI bare metals is docker *)